taipy/common/config/config.pyi

```diff
@@ -508,8 +508,8 @@ class Config:
                 function.
             write_kwargs (Optional[dict]): Additional parameters passed to the
                 `pandas.DataFrame.write_parquet()` function.<br/>
-                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence**
-                than the top-level parameters which are also passed to Pandas.
+                The parameters in *read_kwargs* and *write_kwargs* have a **higher precedence** than the
```

Comment:
@FlorianJacta, I think this is not required. Maybe you changed it by mistake?

Reply:
@FlorianJacta, was just wondering why you made a change here?

Reply:
This is an automatic change not related to the actual changes I have made